server/dao/client.js

```diff
@@ -68,3 +70,24 @@ exports.createClient = (db, fullName, phoneNumber, email, address, wallet) => {
     wallet: wallet,
   });
 };
+
+exports.signupClient = async (
+  db,
+  fullName,
+  phoneNumber,
+  email,
+  password,
+  address,
+  wallet
+) => {
+  const hashedPwd = await bcrypt.hash(password, saltRounds);
```

Comment:
probably is better if you hash password not here but before. in the handler. Not sure. You can search the best practice

Reply:
@G-Salviati Yes, you are right.but I tried to do it before, it took like 3 hours but I couldn't solve it, then I did this.